payments-core/src/main/java/com/stripe/android/view/CardInputWidget.kt

```diff
@@ -1268,6 +1273,7 @@ class CardInputWidget @JvmOverloads constructor(
         private const val STATE_CARD_VIEWED = "state_card_viewed"
         private const val STATE_SUPER_STATE = "state_super_state"
         private const val STATE_POSTAL_CODE_ENABLED = "state_postal_code_enabled"
+        private const val STATE_ON_BEHALF_OF = "state_on_behalf_of"
```

Comment:
is there a good way to share more code between CardInputWidget and CardFormView? It looks like you did some careful thinking about how OBO should work and it'd be great to share that logic

Reply:
and CardMultilineWidget too

Reply:
This was initially done in the view model but was problematic with how `SavedStateRegistry` made to work with this view and the requirements need to restore it with `extras.createSavedStateHandle`. We might be able to share the key across all three but I think we might still have to have every view restore it in some way unless we have a superclass that handles OBO.

Reply:
hmm ok, maybe it's not worth it then